dist/tic-tac-toe.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+/* Produces a diagram that looks like this:
+   1   2   3
+A    |   |
+  -----------
+B    |   |
+  -----------
+C    |   |
+*/
+function getInitialBoard() {
```

Comment:
the function could have `H` and `B` as input parameters to make it easily scale-able

Reply:
and if the `H` and `B` is going to be always same could be used only one param like size

Reply:
I am not sure but it seems like its possible to simplify this function
to something like:
```
for(i = 0; i < size; i++) {
    for (j = 0; j < size; j++) {
        if (i > 0 && j > 0) {
            // print the value and separator
        }
    }
    // here print line
}
```